sdk-api-src/content/winusb/nf-winusb-winusb_writeisochpipeasap.md

```diff
@@ -101,10 +101,11 @@ If the caller sets <i>ContinueStream</i> to TRUE, The transfer fails if Winusb.s
 <b>WinUsb_WriteIsochPipeAsap</b> allows the USB driver stack to choose the starting frame number for the transfer.  If one or more transfers are already pending on the endpoint, the transfer will be scheduled for the frame number immediately following the last frame number of the last currently pending transfer.
 
 
-<b>WinUsb_WriteIsochPipeAsap</b> packetizes the transfer buffer so that in each interval,  the host can send the maximum bytes allowed per interval. The maximum bytes is as specified by the endpoint descriptor for full and high-speed endpoints, and endpoint companion descriptor for SuperSpeed endpoints.
+<b>WinUsb_WriteIsochPipeAsap</b> packetizes the transfer buffer so that in each 1ms interval,  the host can send the maximum bytes allowed per interval. The maximum bytes is as specified by the endpoint descriptor for full and high-speed endpoints, and endpoint companion descriptor for SuperSpeed endpoints.
 If the caller submits multiple write requests to stream data to the device, the transfer size should be a multiple of the maximum bytes per interval (as returned by <a href="https://docs.microsoft.com/windows/desktop/api/winusb/nf-winusb-winusb_querypipeex">WinUsb_QueryPipeEx</a>) * 8 / interval.
 
 
+Because of the transfer packaging, <b>only full 1ms USB frames are available</b> and low-latency applications requiring microframe transfers must use a kernel mode driver instead.
```

Comment:
Same here.

Reply:
```suggestion
Because of the transfer packaging used in the underlying kernel-mode interface, the lowest latency notification to an application or driver is 1ms intervals. This also applies to high-speed/SuperSpeed devices regardless of the interval specified by the endpoint. Applications requiring sub-millisecond latency must implement a custom kernel-mode driver.
```